app/controllers/root_controller.rb

```diff
@@ -16,6 +16,14 @@ def accessibility_statement; end
 
 private
 
+  def authenticate_user!
+    if request.fullpath.bytesize > ActionDispatch::Cookies::MAX_COOKIE_SIZE / 2
+      raise "Request path exceeds maximum length"
```

Comment:
Will this still be logged to Sentry?

Reply:
Yeah. My plan was that we then suppress it, but is there a code-based way not to send it to Sentry?